packages/libs/ui/nft-react-hooks/src/lib/hooks/useCollection/useCollection.ts

```diff
@@ -1,4 +1,4 @@
-import { useQuery } from '@apollo/client';
+import { useCustomQuery } from '../../utils/useCustomQuery';
```

Comment:
i *think* we should avoid relative imports

Reply:
agreed, was putting it off to not do the set up on this pr and increase scope, but after a quick test, it is a one line change on the tsconfig, so included on 01da309!

Reply:
nicely done :)